ui/app/templates/components/generate-credentials-database.hbs

```diff
@@ -5,16 +5,14 @@
 
 <PageHeader as |p|>
   <p.top>
-    <nav class="breadcrumb" aria-label="breadcrumbs">
-      <ul>
-        <li>
-          <span class="sep">&#x0002f;</span>
-          <LinkTo @route="vault.cluster.secrets.backend.overview" @model={{@backendPath}}>
-            {{@backendPath}}
-          </LinkTo>
-        </li>
-      </ul>
-    </nav>
+    <Hds::Breadcrumb>
+      <Hds::Breadcrumb::Item
+        @text={{@backendPath}}
+        @route="vault.cluster.secrets.backend.overview"
+        @model={{@backendPath}}
+      />
+      <Hds::Breadcrumb::Item @text={{@roleName}} @current={{true}} />
```

Comment:
In places where it was obvious, I added a `@current` breadcrumb that matches the h1 tag